apps/workplace/tsconfig.app.json

```diff
@@ -2,7 +2,7 @@
     "extends": "./tsconfig.json",
     "compilerOptions": {
         "outDir": "../../dist/out-tsc",
-        "types": [],
+        "types": ["google.maps"],
```

Comment:
Are you able to add these types to the root tsconfig instead of just this app?

Reply:
I've updated this now.